.github/packaging/project/install_deps.sh

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env bash
+set -xeuo pipefail
+
+function install_deps_debian11() {
+  apt -y install ruby-rubygems make rpm git snapd curl binutils
```

Comment:
https://manpages.org/apt/8 says:

> All features of [apt](https://manpages.org/apt/8)(8) are available in dedicated APT tools like [apt-get](https://manpages.org/apt-get/8)(8) and [apt-cache](https://manpages.org/apt-cache/8)(8) as well. [apt](https://manpages.org/apt/8)(8) just changes the default value of some options (see [apt.conf](https://manpages.org/apt.conf/5)(5) and specifically the Binary scope). So you should prefer using these commands (potentially with some additional options enabled) in your scripts as they keep backward compatibility as much as possible.

Basically `apt` if a user interface and the output may change version to version while `apt-get` is expected to remain the same for script compatibility.